client/http/client.go

```diff
@@ -493,6 +679,56 @@ func (c *client) GetStores(ctx context.Context) (*StoresInfo, error) {
 	return &stores, nil
 }
 
+// GetStoresByState gets the stores info by state.
+func (c *client) GetStoresByState(ctx context.Context, state metapb.StoreState) (*StoresInfo, error) {
+	var stores StoresInfo
+	err := c.requestWithRetry(ctx,
+		"GetStoresByState", StoresByState(state),
+		http.MethodGet, http.NoBody, &stores)
+	if err != nil {
+		return nil, err
+	}
+	return &stores, nil
+}
+
+func (c *client) GetStore(ctx context.Context, storeID uint64) (*StoreInfo, error) {
+	storeInfo := &StoreInfo{}
+	err := c.requestWithRetry(ctx,
+		"GetStore", StoreByID(storeID),
+		http.MethodGet, http.NoBody, storeInfo)
+	if err != nil {
+		return nil, err
+	}
+	return storeInfo, nil
+}
+
+func (c *client) SetStoreLabels(ctx context.Context, storeID uint64, labels map[string]string) (bool, error) {
+	data, err := json.Marshal(labels)
+	if err != nil {
+		return false, errors.Trace(err)
+	}
+	err = c.requestWithRetry(ctx,
+		"SetStoreLabels", StoreLabelByID(storeID),
+		http.MethodPost, bytes.NewBuffer(data), nil)
+	if err != nil {
+		return false, err
+	}
+	return true, nil
+}
+
+func (c *client) DeleteStore(ctx context.Context, storeID uint64) error {
```

Comment:
It's better to group the Interface by severity. some interfaces are only used in operators and it's critical. Once user use this function, should let the user know it's critical when using that. like the group the op interface to the `MaintianceOpClient` etc.